ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/response/FullAdmissionResponseMapper.java

```diff
@@ -54,9 +57,10 @@ public CCDFullAdmissionResponse to(FullAdmissionResponse fullAdmissionResponse)
     @Override
     public FullAdmissionResponse from(CCDFullAdmissionResponse ccdFullAdmissionResponse) {
         CCDYesNoOption ccdFreeMediation = ccdFullAdmissionResponse.getFreeMediationOption();
+        CCDYesNoOption moreTimeNeeded = ccdFullAdmissionResponse.getMoreTimeNeededOption();
         return new FullAdmissionResponse(
             ccdFreeMediation != null ? YesNoOption.valueOf(ccdFreeMediation.name()) : null,
-            YesNoOption.valueOf(ccdFullAdmissionResponse.getMoreTimeNeededOption().name()),
+            moreTimeNeeded != null ? YesNoOption.valueOf(moreTimeNeeded.name()) : null,
```

Comment:
Not sure about Business flow. Should we default to NO if its null or value is not available. The YES, NO and NULL will become three options then

Reply:
we can't default to `NO`, as for full admission we don't have this question asked at all.